website/docs/getting-started/installation.md

```diff
@@ -2,3 +2,81 @@
 title: Installation
 sidebar_position: 4
 ---
+
+odo can be used as a CLI tool and as an IDE plugin; it can be run on Linux, Windows and Mac systems.
+
+## CLI Binary installation
+
+### Installing odo on Linux
+```shell
+  $ curl -L https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-linux-amd64 -o /usr/local/bin/odo
```

Comment:
I think we can auto-detect the system and machine type with `uname`, and get only one instruction for all Linuxes and mac.

An example from https://krew.sigs.k8s.io/docs/user-guide/setup/install/:

```
OS="$(uname | tr '[:upper:]' '[:lower:]')"
ARCH="$(uname -m | sed -e 's/x86_64/amd64/' -e 's/\(arm\)\(64\)\?.*/\1\2/' -e 's/aarch64$/arm64/')"
```

Also, non root user will get an error on trying to write the downloaded file into `/usr/local/bin`. You could instead run (`install` will position the `x` flag correctly):

```
curl ... -o odo
sudo install odo /usr/local/bin/
```